docker/latest/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM nvidia/cuda:10.1-cudnn7-devel
```

Comment:
Perhaps move this file `docker/2.3.0/Dockerfile`. That way we can maintain docker files of different releases as we make thiem